# Awesome AI Meets Bioinformatics&Biomedicine [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
[![GitHub Stars](https://img.shields.io/github/stars/Webioinfo01/Awesome-AI-Meets-Bioinformatics_Biomedicine?style=social)](https://github.com/Webioinfo01/Awesome-AI-Meets-Biomedicine) [![visitors](https://visitor-badge.laobi.icu/badge?page_id=Webioinfo01.Awesome-AI-Meets-Bioinformatics_Biomedicine)](https://github.com/Webioinfo01/Awesome-AI-Meets-Bioinformatics_Biomedicine)

AI separated into "Foundation models" and "AI Agents" for biomedical/bioinformatics/biology research.
## AI Agents

| Year    | Title                                                        | Team                  | Team Website                                                 | Domain                                                     | Venue                | Paper/Source                                                 | Code/Product                                                |
| ------- | ------------------------------------------------------------ | --------------------- | ------------------------------------------------------------ | ---------------------------------------------------------- | -------------------- | ------------------------------------------------------------ | ----------------------------------------------------------- |
| 2025.06 | **NVIDIA Biomedical AI-Q Research Agent Developer Blueprint** | NVIDIA                | [Link](https://github.com/NVIDIA)                            | Drug Discovery (Target Identification)                     | Nvidia's blog        | [Link](https://build.nvidia.com/nvidia/biomedical-aiq-research-agent) | [Link](https://github.com/NVIDIA/bionemo-framework)         |
| 2025.06 | **OriGene: A Self-Evolving Virtual Disease Biologist Automating Therapeutic Target Discovery** |                       |                                                              | General Biomedical Research (therapeutic targets)          | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2025.06.03.657658v1) | [Link](https://gentel-lab.github.io/OriGene-Homepage/)      |
| 2025.06 | **Agent Laboratory: Using LLM Agents as Research Assistants** |                       |                                                              | General Scientific Research                                | arXiv                | [Link](https://arxiv.org/abs/2501.04227)                     | [Link](https://agentlaboratory.github.io/)                  |
| 2025.06 | **CellVoyager: AI CompBio Agent Generates New Insights by Autonomously Analyzing Biological Data** | James Zou             | [Link](https://github.com/zou-group)                         | single cell RNA                                            | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2025.06.03.657517v1) | [Link](https://github.com/zou-group/CellVoyager)            |
| 2025.05 | **Alita: Generalist Agent Enabling Scalable Agentic Reasoning with Minimal Predefinition and Maximal Self-Evolution** | Mengdi Wang           | [Link](https://mwang.princeton.edu/)                         | General Scientific Research(self-evolution and create MCP) | bioRxiv              | [Link](https://arxiv.org/abs/2505.20286)                     | [Link](https://github.com/CharlesQ9/Alita)                  |
| 2025.05 | **BioOmni: A General-Purpose AI Agent for Automated Biomedical Research** | Marinka Zitnik        | [Link](https://zitniklab.hms.harvard.edu/)                   | General Biomedical Research (Multi-modal)                  | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2025.05.30.656746v1) | [Link](https://biomni.stanford.edu/)                        |
| 2025.04 | **The AI Scientist-v2: Workshop-Level Automated Scientific Discovery via Agentic Tree Search** | Sakana AI(David Ha)   | [Link](https://github.com/SakanaAI)                          | General Scientific Research                                | ICLR 2025            | [Link](https://arxiv.org/abs/2504.08066)                     | [Link](https://github.com/SakanaAI/AI-Scientist-v2)         |
| 2025.04 | **SpatialAgent: An autonomous AI agent for spatial biology** | Genentech(Aviv Regev) | [Link](https://github.com/Genentech)                         | Spatial scRNA                                              | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2025.04.03.646459v1) | [Link](https://github.com/Genentech/SpatialAgent)           |
| 2025.03 | **TxAgent: An AI Agent for Therapeutic Reasoning Across a Universe of Tools** | Marinka Zitnik        | [Link](https://zitniklab.hms.harvard.edu/)                   | Personalized Treatment Recommendations                     | arXiv                | [Link](https://arxiv.org/abs/2503.10970)                     | [Link](https://github.com/mims-harvard/TxAgent)             |
| 2025.03 | **DrBioRight 2.0: an LLM-powered bioinformatics chatbot for large-scale cancer functional proteomics analysis** |                       |                                                              | proteomics                                                 | Nature Communication | [Link](https://www.nature.com/articles/s41467-025-57430-4)   | [Link](https://drbioright.org/)                             |
| 2025.03 | **DrugAgent: Automating AI-aided Drug Discovery Programming through LLM Multi-Agent Collaboration** |                       |                                                              | Drug Discovery                                             | arXiv                | [Link](https://arxiv.org/abs/2411.15692)                     | [Link](https://anonymous.4open.science/r/drugagent-5C42/)   |
| 2025.02 | **LIDDIA: Language-based Intelligent Drug Discovery Agent**  |                       |                                                              | Drug Discovery                                             | arXiv                | [Link](https://arxiv.org/abs/2502.13959)                     |                                                             |
| 2025.02 | **Towards an AI co-scientist**                               |                       |                                                              | General Scientific Research                                | arXiv                | [Link](https://arxiv.org/pdf/2502.18864v1)                   |                                                             |
| 2025.01 | **InstructCell: A Multi-Modal AI Copilot for Single-Cell Analysis with Instruction Following** |                       |                                                              | single cell RNA                                            | arXiv                | [Link](https://arxiv.org/abs/2501.08187)                     | [Link](https://github.com/zjunlp/InstructCell)              |
| 2025.01 | **BioAgents: Democratizing Bioinformatics Analysis with Multi-Agent Systems** |                       |                                                              | Multi-omics Pipelines                                      | arXiv                | [Link](https://arxiv.org/abs/2501.06314)                     |                                                             |
| 2024.11 | **BioDiscoveryAgent: An AI Agent for Designing Genetic Perturbation Experiments** |                       |                                                              | Experimental Design (Genetic Perturbation)                 | arXiv                | [Link](https://arxiv.org/abs/2405.17631)                     | [Link](https://github.com/snap-stanford/BioDiscoveryAgent/) |
| 2024.11 | **The Virtual Lab: AI Agents Design New SARS-CoV-2 Nanobodies with Experimental Validation** | James Zou             | [Link](https://github.com/zou-group)                         | General Scientific Research                                | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2024.11.11.623004v1) | [Link](https://github.com/zou-group/virtual-lab)            |
| 2024.10 | **An AI Agent for Fully Automated Multi-Omic Analyses**      |                       |                                                              | Multi-omics Pipelines                                      | Advanced Science     | [Link](https://advanced.onlinelibrary.wiley.com/doi/10.1002/advs.202407094) | [Link](https://github.com/JoshuaChou2018/AutoBA)            |
| 2024.09 | **The AI Scientist: Towards Fully Automated Open-Ended Scientific Discovery** | Sakana AI(David Ha)   | [Link](https://github.com/SakanaAI)                          | General Scientific Research                                | arXiv                | [Link](https://arxiv.org/abs/2408.06292)                     | [Link](https://github.com/SakanaAI/AI-Scientist)            |
| 2024.07 | **CellAgent: An LLM-driven Multi-Agent Framework for Automated Single-cell Data Analysis** |                       |                                                              | single cell RNA                                            | arXiv                | [Link](https://arxiv.org/abs/2407.09811)                     | [Link](http://cell.agent4science.cn/)                       |
| 2024.05 | **A Data-Intelligence-Intensive Bioinformatics Copilot System for Large-scale Omics Researches and Scientific Insights** |                       |                                                              | Multi-omics/single cell RNA                                | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2024.05.19.594895v2) | [Link](https://github.com/lyyang01/Bio-Copilot)             |
| 2024.04 | **CRISPR-GPT: An LLM Agent for Automated Design of Gene-Editing Experiments** | Mengdi Wang, Le Cong  | [Wang lab](https://mwang.princeton.edu/), [Cong lab](https://github.com/cong-lab) | Experimental Design (CRISPR Genetic Editing)               | bioRxiv              | [Link](https://www.biorxiv.org/content/10.1101/2024.04.25.591003v3) | [Link](https://github.com/cong-lab/crispr-gpt-pub)          |
| 2024.01 | **ProtAgents: Protein discovery via large language model multi-agent collaborations combining physics and machine learning** |                       |                                                              | Protein design and discovery                               | arXiv                | [Link](https://arxiv.org/abs/2402.04268)                     | [Link](https://github.com/lamm-mit/ProtAgents)              |

## Foundation models

| Year    | Title                                                        | Team                                         | Team Website                                  | Domain                                                       | Venue                       | Paper/ Source                                                | Code/Product                                                 |
| ------- | ------------------------------------------------------------ | -------------------------------------------- | --------------------------------------------- | ------------------------------------------------------------ | --------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 2025.07 | **Spatia: Multimodal Model for Prediction and Generation of Spatial Cell Phenotypes** | Marinka Zitnik                               | [Link](https://zitniklab.hms.harvard.edu/)    | spatial single cell RNA                                      | ArXiv                       | [Link](https://arxiv.org/html/2507.04704v1#S4)               |                                                              |
| 2025.07 | **RiNALMo: general-purpose RNA language models can generalize well on structure prediction tasks** |                                              |                                               | RNA                                                          | Nature Communication        | [Link](https://www.nature.com/articles/s41467-025-60872-5)   | [Link](https://github.com/lbcb-sci/RiNALMo)                  |
| 2025.06 | **Predicting cellular responses to perturbation across diverse contexts with State** | Arc Institute(Yusuf H. Roohani)              | [Link](https://github.com/ArcInstitute)       | genetic, signaling, and chemical perturbation scRNA          | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661135v1) | [Link](https://github.com/ArcInstitute/state)                |
| 2025.06 | **AlphaGenome: Advancing regulatory variant effect prediction with a unified DNA sequence model** | Google DeepMind(Pushmeet Kohli)              | [Link](https://github.com/google-deepmind)    | DNA/RNA                                                      | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.06.25.661532v1) | [Link](https://github.com/google-deepmind/alphagenome)       |
| 2025.06 | **SPACE: Your Genomic Profile Predictor is a Powerful DNA Foundation Model** |                                              |                                               | DNA                                                          | ICML 2025                   | [Link](https://arxiv.org/abs/2506.01833)                     | [Link](https://github.com/YangLab/CSI)                       |
| 2025.06 | **UniCure: A Foundation Model for Predicting Personalized Cancer Therapy Response** |                                              |                                               | drug perturbation scRNA                                      | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.06.14.658531v1) | [Link](https://github.com/ZexiChen502/UniCure)               |
| 2025.06 | **A multimodal conversational agent for DNA, RNA and protein tasks** | InstaDeep(Thomas Pierrot)                    | [Link](https://huggingface.co/InstaDeepAI)    | Diverse Omics include DNA/RNA/Protein                        | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-025-01047-1)   | [Link](https://huggingface.co/InstaDeepAI/ChatNT)            |
| 2025.05 | **A visual–omics foundation model to bridge histopathology with spatial transcriptomics** | Guangyu Wang                                 | [Link](https://guangyuwanglab.github.io/web/) | histopathology and spatial single cell RNA                   | Nature Methods              | [Link](https://www.nature.com/articles/s41592-025-02707-1)   | [Link](https://github.com/GuangyuWangLab2021/Loki)           |
| 2025.05 | **GeneBreaker: Jailbreak Attacks against DNA Language Models with Pathogenicity Guidance** | Mengdi Wang                                  | [Link](https://mwang.princeton.edu/)          | Biosafety for DNA                                            | arXiv                       | [Link](https://arxiv.org/abs/2505.23839)                     | [Link](https://github.com/zaixizhang/GeneBreaker)            |
| 2025.05 | **BioReason: Incentivizing Multimodal Biological Reasoning within a DNA-LLM Model** | Bo Wang                                      | [Link](https://github.com/bowang-lab)         | DNA                                                          | arXiv                       | [Link](https://arxiv.org/abs/2505.23579)                     | [Link](https://github.com/bowang-lab/BioReason)              |
| 2025.05 | **CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells** |                                              |                                               | single cell RNA                                              | Nature Communications       | [Link](https://www.nature.com/articles/s41467-025-59926-5)   | [Link](https://github.com/biomed-AI/CellFM)                  |
| 2025.04 | **Scaling Large Language Models for Next-Generation Single-Cell Analysis** | David van Dijk                               | [Link](https://github.com/vandijklab)         | single cell RNA                                              | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.04.14.648850v1) | [Link](https://github.com/vandijklab/cell2sentence)          |
| 2025.04 | **CellFlow enables generative single-cell phenotype modeling with flow matching** | Fabian J. Theis                              | [Link](https://github.com/theislab)           | Perturbation single cell RNA                                 | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.04.11.648220v1) | [Link](https://github.com/theislab/CellFlow)                 |
| 2025.02 | **Genome modeling and design across all domains of life with Evo 2** | Arc Institute(Brian L. Hie)                  | [Link](https://github.com/ArcInstitute)       | DNA                                                          | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.02.18.638918v1) | [Link](https://github.com/arcinstitute/evo2)                 |
| 2025.02 | **GENERator: A Long-Context Generative Genomic Foundation Model** |                                              |                                               | DNA                                                          | arXiv                       | [Link](https://arxiv.org/abs/2502.07272)                     | [Link](https://github.com/GenerTeam/GENERator)               |
| 2025.02 | **scGPT-spatial: Continual Pretraining of Single-Cell Foundation Model for Spatial Transcriptomics** | Bo Wang                                      | [Link](https://github.com/bowang-lab)         | spatial single cell RNA                                      | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2025.02.05.636714v1) | [Link](https://github.com/bowang-lab/scGPT-spatial)          |
| 2025.01 | **A foundation model of transcription across human cell types** |                                              |                                               | transcriptional regulation(ATAC-seq)                         | Nature                      | [Link](https://www.nature.com/articles/s41586-024-08391-z)   | [Link](https://github.com/GET-Foundation)                    |
| 2025.01 | **GENA-LM: a new DNA language model for long sequences**     |                                              |                                               | DNA                                                          | Nucleic Acids Research      | [Link](https://academic.oup.com/nar/article/53/2/gkae1310/7954523) | [Link](https://github.com/AIRI-Institute/GENA_LM)            |
| 2025.01 | **Simulating 500 million years of evolution with a language model** | EvolutionaryScale(Alexander Rives)           | [Link](https://github.com/evolutionaryscale)  | Protein                                                      | Science                     | [Link](https://www.science.org/doi/10.1126/science.ads0018)  | [Link](https://github.com/AIRI-Institute/GENA_LM)            |
| 2024.11 | **Nucleotide Transformer: building and evaluating robust foundation models for human genomics** |                                              |                                               | DNA                                                          | Nature Methods              | [Link](https://www.nature.com/articles/s41592-024-02523-z)   | [Link](https://github.com/instadeepai/nucleotide-transformer) |
| 2024.10 | **Orthrus: Towards Evolutionary and Functional RNA Foundation Models** |                                              |                                               | RNA                                                          | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2024.10.10.617658v1) | [Link](https://github.com/bowang-lab/Orthrus)                |
| 2024.09 | **stFormer: a foundation model for spatial transcriptomics** |                                              |                                               | DNA                                                          | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2024.09.27.615337v6) | [Link]([Link](https://github.com/csh3/stFormer))             |
| 2024.08 | **BioRAG: A RAG-LLM Framework for Biological Question Reasoning** |                                              |                                               | Life science question answer (RAG)                           | arXiv                       | [Link](https://arxiv.org/abs/2408.01107)                     |                                                              |
| 2024.07 | **DNA language model GROVER learns sequence context in the human genome** |                                              |                                               | DNA                                                          | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-024-00872-0)   | [Link](https://zenodo.org/records/13374192)                  |
| 2024.06 | **scFoundation: Large-scale foundation model on single-cell transcriptomics** | Biomap research(Le Song, Xuegong Zhang)      | [Link](https://github.com/biomap-research)    | single cell RNA                                              | Nature Communication        | [Link](https://www.nature.com/articles/s41592-024-02305-7)   | [Link](https://github.com/biomap-research/scFoundation)      |
| 2024.06 | **Improving Medical Reasoning through Retrieval and Self-Reflection with Retrieval-Augmented Large Language Models** |                                              |                                               | biomedical question answer (RAG)                             | ISMB 24                     | [Link](https://arxiv.org/abs/2401.15269)                     | [Link](https://github.com/dmis-lab/self-biorag)              |
| 2024.06 | **Multi-modal Transfer Learning between Biological Foundation Models** | InstaDeep(Guillaume Richard, Thomas Pierrot) | [Link](https://huggingface.co/InstaDeepAI)    | Diverse Omics include DNA/RNA/Protein                        | arXiv                       | [Link](https://arxiv.org/abs/2406.14150)                     | [Link](https://huggingface.co/InstaDeepAI/isoformer)         |
| 2024.05 | **Accurate structure prediction of biomolecular interactions with AlphaFold 3** | Google DeepMind(John M. Jumper)              | [Link](https://github.com/google-deepmind)    | Protein                                                      | Nature                      | [Link](https://www.nature.com/articles/s41586-024-07487-w)   | [Link](https://github.com/google-deepmind/alphafold3)        |
| 2024.03 | **Caduceus: Bi-Directional Equivariant Long-Range DNA Sequence Modeling** |                                              |                                               | DNA                                                          | arXiv                       | [Link](https://arxiv.org/abs/2403.03234)                     | [Link](https://caduceus-dna.github.io/)                      |
| 2024.03 | **Nicheformer: a foundation model for single-cell and spatial omics** |                                              |                                               | single-cell and spatial omics                                | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2024.04.15.589472v1) | [Link](https://github.com/theislab/nicheformer)              |
| 2024.02 | **scGPT: toward building a foundation model for single-cell multi-omics using generative AI** | Bo Wang                                      | [Link](https://github.com/bowang-lab)         | single cell RNA                                              | Nature Methods              | [Link](https://www.nature.com/articles/s41592-024-02201-0)   | [Link](https://github.com/bowang-lab/scGPT)                  |
| 2024.01 | **OmniNA: A foundation model for nucleotide sequences**      |                                              |                                               | DNA/RNA                                                      | bioRxiv                     | [Link](https://www.biorxiv.org/content/10.1101/2024.01.14.575543v1) |                                                              |
| 2024.01 | **xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein** | Biomap research(Le Song)                     | [Link](https://github.com/biomap-research)    | Protein                                                      | arXiv                       | [Link](https://arxiv.org/abs/2401.06199)                     | [Link](https://github.com/biomap-research/xTrimoPGLM)        |
| 2024.01 | **Geneverse: A Collection of Open-source Multimodal Large Language Models for Genomic and Proteomic Research** |                                              |                                               | DNA/Protein                                                  | EMNLP 2024                  | [Link](https://openreview.net/forum?id=g1NplcWH7X)           | [Link](https://github.com/HelloWorldLTY/Geneverse)           |
| 2023.11 | **HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution** |                                              |                                               | DNA                                                          | NeurIPS 2023                | [Link](https://papers.neurips.cc/paper_files/paper/2023/file/86ab6927ee4ae9bde4247793c46797c7-Paper-Conference.pdf) | [Link](https://github.com/HazyResearch/hyena-dna)            |
| 2023.06 | **DNABERT-2: Efficient Foundation Model and Benchmark For Multi-Species Genome** |                                              |                                               | DNA                                                          | ICLR 2024                   | [Link](https://www.nature.com/articles/s41586-023-06139-9)   | [Link](https://arxiv.org/abs/2306.15006)                     |
| 2023.05 | **Transfer learning enables predictions in network biology** |                                              |                                               | General scRNA tasks, discovery of key network regulators and candidate therapeutic targets | Nature                      | [Link](https://www.nature.com/articles/s41586-023-06139-9)   | [Link](https://huggingface.co/ctheodoris/Geneformer)         |
| 2021.07 | **DNABERT: pre-trained Bidirectional Encoder Representations from Transformers for DNA-language** |                                              |                                               | DNA                                                          | Bioinformatics              | [Link](https://academic.oup.com/bioinformatics/article/37/15/2112/6128680) | [Link](https://github.com/jerryji1993/DNABERT)               |
|         |                                                              |                                              |                                               |                                                              |                             |                                                              |                                                              |

## AI Tools

| Year    | Title                                                        | Domain                                 | Venue          | Paper/Source                                                 | Code/Product                                           |
| ------- | ------------------------------------------------------------ | -------------------------------------- | -------------- | ------------------------------------------------------------ | ------------------------------------------------------ |
| 2025.07 | **GPTBioInsightor**                                          | Single Cell annotation                 |                |                                                              | [Link](https://github.com/huang-sh/GPTBioInsightor)    |
| 2025.06 | **scExtract: leveraging large language models for fully automated single-cell RNA-seq data annotation and prior-informed multi-dataset integration** | scRNA downstream tasks                 | Genome Biology | [Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03639-x) | [Link](https://github.com/yxwucq/scExtract)            |
| 2025.04 | **Large Language Model Consensus Substantially Improves the Cell Type Annotation Accuracy for scRNA-seq Data** | Single Cell annotation                 | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2025.04.10.647852v1) | [Link](https://github.com/cafferychen777/mLLMCelltype) |
| 2025.04 | **SCassist: An AI Based Workflow Assistant for Single-Cell Analysis** | Single Cell tasks(cluster, annotation) | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2025.04.22.650107v1) | [Link](https://github.com/NIH-NEI/SCassist)            |
| 2024.12 | **CASSIA: a multi-agent large language model for reference free, interpretable, and automated cell annotation of single-cell RNA-sequencing data** | Single Cell annotation                 | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2024.12.04.626476v2) | [Link](https://github.com/ElliotXie/CASSIA)            |

## Databases/Simulation

| Year    | Title                                                        | Team                              | Team Website                               | Domain                                                       | Venue          | Paper/Source                                                 | Code/Data                                                    |
| ------- | ------------------------------------------------------------ | --------------------------------- | ------------------------------------------ | ------------------------------------------------------------ | -------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 2025.08 | **Drug Response Omics association MAp (DROMA, 卓玛)**        |                                   |                                            | Biomarker discovery, Drug prediction,(18 projects with 2,600+ samples and 56,000+ drugs, with full ecosystem) |                |                                                              | [Link](https://github.com/mugpeng/DROMA)                     |
| 2025.06 | **X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models** |                                   |                                            | Genome-wide Perturb-seq Datasets                             | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2025.06.11.659105v1) | [Link](https://doi.org/10.25452/figshare.plus.29190726)      |
| 2025.04 | **scMultiSim: simulation of single-cell multi-omics and spatial data guided by gene regulatory networks and cell–cell interactions** |                                   |                                            | Simulation of single-cell multi-omics and spatial data       | Nature Methods | [Link](https://www.nature.com/articles/s41592-025-02651-0)   | [Link](https://github.com/ZhangLabGT/scMultiSim/)            |
| 2025.02 | **scBaseCount: an AI agent-curated, uniformly processed, and continually expanding single cell data repository** | Arc Institute(Yusuf H. Roohani)   | [Link](https://github.com/ArcInstitute)    | preprocess scRNA data                                        | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2025.02.27.640494v2) | [Link](https://github.com/ArcInstitute/SRAgent)              |
| 2024.06 | **Signals in the Cells: Multimodal and Contextualized Machine Learning Foundations for Therapeutics** | Marinka Zitnik                    | [Link](https://zitniklab.hms.harvard.edu/) | Drug Discovery (Therapeutic science, 66 datasets)            | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2024.06.12.598655v3) | [Link](https://tdcommons.ai/)                                |
| 2025.02 | **Tahoe-100M: A Giga-Scale Single-Cell Perturbation Atlas for Context-Dependent Gene Function and Cellular Modeling** | tahoebio(but in arc's repository) | [Link](https://github.com/ArcInstitute)    | Drug perturbation scRNA                                      | bioRxiv        | [Link](https://www.biorxiv.org/content/10.1101/2025.02.20.639398v3) | [Link](https://github.com/ArcInstitute/arc-virtual-cell-atlas) |
| 2024.01 | **scPerturb: harmonized single-cell perturbation data**      |                                   |                                            | scRNA perturbation(44 public datasets, drugs and genes)      | Nature Methods | [Link](https://www.nature.com/articles/s41592-023-02144-y)   | [Link](https://github.com/sanderlab/scPerturb/)              |

## Benchmarks

| Year    | Title                                                        | Team                            | Team Website                            | Domain                            | Venue        | Paper/Source                                                 | Code/Data                                         |
| ------- | ------------------------------------------------------------ | ------------------------------- | --------------------------------------- | --------------------------------- | ------------ | ------------------------------------------------------------ | ------------------------------------------------- |
| 2025.06 | **Fundamental Limitations of Foundation Models in Single-Cell Transcriptomics** |                                 |                                         | evaluate cell-type classification | bioRxiv      | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661767v1) |                                                   |
| 2025.06 | **Cell-Eval(Predicting cellular responses to perturbation across diverse contexts with State)** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) | Perturbation scRNA                | bioRxiv      | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661135v1) | [Link](https://github.com/ArcInstitute/cell-eval) |
| 2024.12 | **PerturBench: Benchmarking Machine Learning Models for Cellular Perturbation Analysis** |                                 |                                         | Perturbation scRNA                | NeurIPS 2024 | [Link](https://neurips.cc/virtual/2024/102911)               | [Link](https://github.com/altoslabs/perturbench)  |
|         |                                                              |                                 |                                         |                                   |              |                                                              |                                                   |

## Reviews

| Year    | Title                                                        | Domain                           | Venue                | Paper/Source                                                 |
| ------- | ------------------------------------------------------------ | -------------------------------- | -------------------- | ------------------------------------------------------------ |
| 2025.05 | **Large Language Models in Bioinformatics: A Survey**        | Bioinformatics foundation models | arXiv                | [Link](https://arxiv.org/abs/2503.04490)                     |
| 2025.04 | **Model Context Protocol (MCP): Landscape, Security Threats, and Future Research Directions** | MCP for agents                   | arXiv                | [Link](https://arxiv.org/abs/2503.23278)                     |
| 2025.04 | **A Survey of the Model Context Protocol (MCP): Standardizing Context to Enhance Large Language Models (LLMs)** | MCP for agents                   | Preprints            | [Link](https://www.preprints.org/manuscript/202504.0245/v1)  |
| 2024.10 | **Empowering biomedical discovery with AI agents**           | Biomedical Agents                | Cell                 | [Link](https://www.cell.com/cell/fulltext/S0092-8674(24)01070-5) |
| 2024.08 | **Transformers in single-cell omics: a review and new perspectives** | Single cell foundation models    | Nature Methods       | [Link](https://www.nature.com/articles/s41592-024-02353-z)   |
| 2024.08 | **Language models for biological research: a primer**        | biological foundation models     | Nature Methods       | [Link](https://www.nature.com/articles/s41592-024-02354-y)   |
| 2024.07 | **Foundation models for bioinformatics**                     | Bioinformatics foundation models | Quantitative Biology | [Link](https://onlinelibrary.wiley.com/doi/10.1002/qub2.69)  |
|         |                                                              |                                  |                      |                                                              |


##  Other Awesome Projects

| Title                                           | Project                                                      |
| ----------------------------------------------- | ------------------------------------------------------------ |
| **Awesome-LLMs-meet-genomes**                   | [Link](https://github.com/ychuest/Awesome-LLMs-meet-genomes) |
| **Awesome-Virtual-Cell**                        | [Link](https://github.com/Boom5426/Awesome-Virtual-Cell)     |
| **Awesome-LLM-Scientific-Discovery**            | [Link](https://github.com/HKUST-KnowComp/Awesome-LLM-Scientific-Discovery) |
| **Awesome LLM Agents for Scientific Discovery** | [Link](https://github.com/zhoujieli/Awesome-LLM-Agents-Scientific-Discovery) |
| **Awesome Papers on Agents for Science**        | [Link](https://github.com/OSU-NLP-Group/awesome-agents4science) |
| **awesome-single-cell**                         | [Link](https://github.com/seandavi/awesome-single-cell)      |
| **Awesome-Bioinformatics**                      | [Link](https://github.com/danielecook/Awesome-Bioinformatics) |
| **awesome**                                     | [Link](https://github.com/sindresorhus/awesome)              |

## Others


# Ref