#  Awesome AI Meets Biology[![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
[![Website](https://img.shields.io/website?url=https%3A//droma01.github.io/DROMA)](https://webioinfo01.github.io/Awesome-AI-Meets-Biology/)
[![GitHub Stars](https://img.shields.io/github/stars/Webioinfo01/Awesome-AI-Meets-Bioinformatics_Biomedicine?style=social)](https://github.com/Webioinfo01/Awesome-AI-Meets-Biomedicine) 
[![visitors](https://visitor-badge.laobi.icu/badge?page_id=Webioinfo01.Awesome-AI-Meets-Bioinformatics_Biomedicine)](https://github.com/Webioinfo01/Awesome-AI-Meets-Bioinformatics_Biomedicine)

**Visit the official website at: https://webioinfo01.github.io/Awesome-AI-Meets-Biology/**

AI separated into "Foundation models" and "AI Agents" for biology/biomedical/bioinformatics research.

## üìã Table of Contents
- [üåü AI Agents](#AI-Agents)
- [üéØ Foundation models](#Foundation-models)
- [üõ†Ô∏è AI Tools](#AI-Tools)
- [üíæ Databases/Simulation](#DatabasesSimulation)
- [üìä Benchmarks](#Benchmarks)
- [üìö Reviews](#Reviews)
- [üîó Other Awesome Projects](#Other-Awesome-Projects)

## AI Agents

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **PromptBio: A Multi-Agent AI Platform for Bioinformatics Data Analysis** | PromptBio(Xiao Yang) | [Link](https://www.promptbio.ai/) |  | multiple agents system for bioinformatics | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.07.05.663295v1) | [Link](https://www.promptbio.ai/download.html) |
| 2025.06 | **NVIDIA Biomedical AI-Q Research Agent Developer Blueprint** | NVIDIA | [Link](https://github.com/NVIDIA) |  | Drug Discovery (Target Identification) | Nvidia's blog | [Link](https://build.nvidia.com/nvidia/biomedical-aiq-research-agent) | [Link](https://github.com/NVIDIA/bionemo-framework) ![GitHub Stars](https://img.shields.io/github/stars/NVIDIA/bionemo-framework) |
| 2025.06 | **OriGene: A Self-Evolving Virtual Disease Biologist Automating Therapeutic Target Discovery** |  |  |  | General Biomedical Research (therapeutic targets) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.03.657658v1) | [Link](https://gentel-lab.github.io/OriGene-Homepage/) ![GitHub Stars](https://img.shields.io/github/stars/GENTEL-lab/OriGene) |
| 2025.06 | **Agent Laboratory: Using LLM Agents as Research Assistants** |  |  |  | General Scientific Research | arXiv | [Link](https://arxiv.org/abs/2501.04227) | [Link](https://agentlaboratory.github.io/) ![GitHub Stars](https://img.shields.io/github/stars/SamuelSchmidgall/AgentLaboratory) |
| 2025.06 | **CellVoyager: AI CompBio Agent Generates New Insights by Autonomously Analyzing Biological Data** | James Zou | [Link](https://github.com/zou-group) |  | single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.03.657517v1) | [Link](https://github.com/zou-group/CellVoyager) ![GitHub Stars](https://img.shields.io/github/stars/zou-group/CellVoyager) |
| 2025.05 | **Alita: Generalist Agent Enabling Scalable Agentic Reasoning with Minimal Predefinition and Maximal Self-Evolution** | Mengdi Wang | [Link](https://mwang.princeton.edu/) |  | General Scientific Research(self-evolution and create MCP) | bioRxiv | [Link](https://arxiv.org/abs/2505.20286) | [Link](https://github.com/CharlesQ9/Alita) ![GitHub Stars](https://img.shields.io/github/stars/CharlesQ9/Alita) |
| 2025.05 | **BioOmni: A General-Purpose AI Agent for Automated Biomedical Research** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | General Biomedical Research (Multi-modal) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.05.30.656746v1) | [Link](https://biomni.stanford.edu/) ![GitHub Stars](https://img.shields.io/github/stars/snap-stanford/biomni) |
| 2025.04 | **The AI Scientist-v2: Workshop-Level Automated Scientific Discovery via Agentic Tree Search** | Sakana AI(David Ha) | [Link](https://github.com/SakanaAI) |  | General Scientific Research | ICLR 2025 | [Link](https://arxiv.org/abs/2504.08066) | [Link](https://github.com/SakanaAI/AI-Scientist-v2) ![GitHub Stars](https://img.shields.io/github/stars/SakanaAI/AI-Scientist-v2) |
| 2025.04 | **SpatialAgent: An autonomous AI agent for spatial biology** | Genentech(Aviv Regev) | [Link](https://github.com/Genentech) |  | Spatial scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.03.646459v1) | [Link](https://github.com/Genentech/SpatialAgent) ![GitHub Stars](https://img.shields.io/github/stars/Genentech/SpatialAgent) |
| 2025.03 | **TxAgent: An AI Agent for Therapeutic Reasoning Across a Universe of Tools** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | Personalized Treatment Recommendations | arXiv | [Link](https://arxiv.org/abs/2503.10970) | [Link](https://github.com/mims-harvard/TxAgent) ![GitHub Stars](https://img.shields.io/github/stars/mims-harvard/TxAgent) |
| 2025.03 | **DrBioRight 2.0: an LLM-powered bioinformatics chatbot for large-scale cancer functional proteomics analysis** |  |  |  | proteomics | Nature Communication | [Link](https://www.nature.com/articles/s41467-025-57430-4) | [Link](https://drbioright.org/) |
| 2025.03 | **DrugAgent: Automating AI-aided Drug Discovery Programming through LLM Multi-Agent Collaboration** |  |  |  | Drug Discovery | arXiv | [Link](https://arxiv.org/abs/2411.15692) | [Link](https://anonymous.4open.science/r/drugagent-5C42/) |
| 2025.02 | **LIDDIA: Language-based Intelligent Drug Discovery Agent** |  |  |  | Drug Discovery | arXiv | [Link](https://arxiv.org/abs/2502.13959) |  |
| 2025.02 | **Towards an AI co-scientist** |  |  |  | General Scientific Research | arXiv | [Link](https://arxiv.org/pdf/2502.18864v1) |  |
| 2025.01 | **BioMaster: Multi-agent System for Automated Bioinformatics Analysis Workflow** |  |  |  | Multi-omics Pipelines | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.01.23.634608v1) | [Link](https://github.com/ai4nucleome/BioMaster) ![GitHub Stars](https://img.shields.io/github/stars/ai4nucleome/BioMaster) |
| 2025.01 | **InstructCell: A Multi-Modal AI Copilot for Single-Cell Analysis with Instruction Following** |  |  |  | single cell RNA | arXiv | [Link](https://arxiv.org/abs/2501.08187) | [Link](https://github.com/zjunlp/InstructCell) ![GitHub Stars](https://img.shields.io/github/stars/zjunlp/InstructCell) |
| 2025.01 | **BioAgents: Democratizing Bioinformatics Analysis with Multi-Agent Systems** | Microsoft Research(Venkat S. Malladi) | [Link](https://github.com/microsoft) |  | Multi-omics Pipelines | arXiv | [Link](https://arxiv.org/abs/2501.06314) |  |
| 2024.11 | **BioDiscoveryAgent: An AI Agent for Designing Genetic Perturbation Experiments** |  |  |  | Experimental Design (Genetic Perturbation) | arXiv | [Link](https://arxiv.org/abs/2405.17631) | [Link](https://github.com/snap-stanford/BioDiscoveryAgent/) ![GitHub Stars](https://img.shields.io/github/stars/snap-stanford/BioDiscoveryAgent) |
| 2024.11 | **The Virtual Lab: AI Agents Design New SARS-CoV-2 Nanobodies with Experimental Validation** | James Zou | [Link](https://github.com/zou-group) |  | General Scientific Research | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.11.11.623004v1) | [Link](https://github.com/zou-group/virtual-lab) ![GitHub Stars](https://img.shields.io/github/stars/zou-group/virtual-lab) |
| 2024.10 | **An AI Agent for Fully Automated Multi-Omic Analyses** |  |  |  | Multi-omics Pipelines | Advanced Science | [Link](https://advanced.onlinelibrary.wiley.com/doi/10.1002/advs.202407094) | [Link](https://github.com/JoshuaChou2018/AutoBA) ![GitHub Stars](https://img.shields.io/github/stars/JoshuaChou2018/AutoBA) |
| 2024.09 | **The AI Scientist: Towards Fully Automated Open-Ended Scientific Discovery** | Sakana AI(David Ha) | [Link](https://github.com/SakanaAI) |  | General Scientific Research | arXiv | [Link](https://arxiv.org/abs/2408.06292) | [Link](https://github.com/SakanaAI/AI-Scientist) ![GitHub Stars](https://img.shields.io/github/stars/SakanaAI/AI-Scientist) |
| 2024.07 | **CellAgent: An LLM-driven Multi-Agent Framework for Automated Single-cell Data Analysis** |  |  |  | single cell RNA | arXiv | [Link](https://arxiv.org/abs/2407.09811) | [Link](http://cell.agent4science.cn/) |
| 2024.05 | **A Data-Intelligence-Intensive Bioinformatics Copilot System for Large-scale Omics Researches and Scientific Insights** |  |  |  | Multi-omics/single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.05.19.594895v2) | [Link](https://github.com/lyyang01/Bio-Copilot) ![GitHub Stars](https://img.shields.io/github/stars/lyyang01/Bio-Copilot) |
| 2024.04 | **CRISPR-GPT: An LLM Agent for Automated Design of Gene-Editing Experiments** | Mengdi Wang, Le Cong | [Link](https://mwang.princeton.edu/) |  | Experimental Design (CRISPR Genetic Editing) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.04.25.591003v3) | [Link](https://github.com/cong-lab/crispr-gpt-pub) ![GitHub Stars](https://img.shields.io/github/stars/cong-lab/crispr-gpt-pub) |
| 2024.01 | **ProtAgents: Protein discovery via large language model multi-agent collaborations combining physics and machine learning** |  |  |  | Protein design and discovery | arXiv | [Link](https://arxiv.org/abs/2402.04268) | [Link](https://github.com/lamm-mit/ProtAgents) ![GitHub Stars](https://img.shields.io/github/stars/lamm-mit/ProtAgents) |

## Foundation models

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **ODFormer: a Virtual Organoid for Predicting Personalized Therapeutic Responses in Pancreatic Cancer** |  |  |  | Organoid drug response prediction | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.07.08.663664v2) | [Link](https://github.com/microsoft/bioemu) ![GitHub Stars](https://img.shields.io/github/stars/xujing363/ODFormer) |
| 2025.07 | **Scalable emulation of protein equilibrium ensembles with generative deep learning** | Microsoft Research(Frank No√©) | [Link](https://github.com/microsoft) |  | Protein | Science | [Link](https://www.science.org/doi/10.1126/science.adv9817) | [Link](https://github.com/microsoft/bioemu) ![GitHub Stars](https://img.shields.io/github/stars/microsoft/bioemu) |
| 2025.07 | **Spatia: Multimodal Model for Prediction and Generation of Spatial Cell Phenotypes** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | spatial single cell RNA | ArXiv | [Link](https://arxiv.org/html/2507.04704v1#S4) |  |
| 2025.07 | **RiNALMo: general-purpose RNA language models can generalize well on structure prediction tasks** |  |  |  | RNA | Nature Communication | [Link](https://www.nature.com/articles/s41467-025-60872-5) | [Link](https://github.com/lbcb-sci/RiNALMo) ![GitHub Stars](https://img.shields.io/github/stars/lbcb-sci/RiNALMo) |
| 2025.06 | **World Models as Simulators of Patient Biology: Oncology Counterfactual Therapeutics Oracle (OCTO)** | Noetik | [Link](https://www.noetik.ai/research) |  | spatial single cell RNA perturbation | noetik's report | [Link](https://www.noetik.ai/octo) |  |
| 2025.06 | **Generalized biological foundation model with unified nucleic acid and protein language** | Alibaba(Zhaorong Li) | [Link](https://github.com/LucaOne) |  | DNA/RNA/Protein | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-025-01044-4) | [Link](https://github.com/LucaOne/LucaOne) ![GitHub Stars](https://img.shields.io/github/stars/LucaOne/LucaOne) |
| 2025.06 | **Predicting cellular responses to perturbation across diverse contexts with State** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) |  | genetic, signaling, and chemical perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661135v1) | [Link](https://github.com/ArcInstitute/state) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/state) |
| 2025.06 | **AlphaGenome: Advancing regulatory variant effect prediction with a unified DNA sequence model** | Google DeepMind(Pushmeet Kohli) | [Link](https://github.com/google-deepmind) |  | DNA/RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.25.661532v1) | [Link](https://github.com/google-deepmind/alphagenome) ![GitHub Stars](https://img.shields.io/github/stars/google-deepmind/alphagenome) |
| 2025.06 | **SPACE: Your Genomic Profile Predictor is a Powerful DNA Foundation Model** |  |  |  | DNA | ICML 2025 | [Link](https://arxiv.org/abs/2506.01833) | [Link](https://github.com/YangLab/CSI) ![GitHub Stars](https://img.shields.io/github/stars/YangLab/CSI) |
| 2025.06 | **UniCure: A Foundation Model for Predicting Personalized Cancer Therapy Response** |  |  |  | drug perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.14.658531v1) | [Link](https://github.com/ZexiChen502/UniCure) ![GitHub Stars](https://img.shields.io/github/stars/ZexiChen502/UniCure) |
| 2025.06 | **A multimodal conversational agent for DNA, RNA and protein tasks** | InstaDeep(Thomas Pierrot) | [Link](https://huggingface.co/InstaDeepAI) |  | Diverse Omics include DNA/RNA/Protein | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-025-01047-1) | [Link](https://huggingface.co/InstaDeepAI/ChatNT) |
| 2025.05 | **A visual‚Äìomics foundation model to bridge histopathology with spatial transcriptomics** | Guangyu Wang | [Link](https://guangyuwanglab.github.io/web/) |  | histopathology and spatial single cell RNA | Nature Methods | [Link](https://www.nature.com/articles/s41592-025-02707-1) | [Link](https://github.com/GuangyuWangLab2021/Loki) ![GitHub Stars](https://img.shields.io/github/stars/GuangyuWangLab2021/Loki) |
| 2025.05 | **GeneBreaker: Jailbreak Attacks against DNA Language Models with Pathogenicity Guidance** | Mengdi Wang | [Link](https://mwang.princeton.edu/) |  | Biosafety for DNA | arXiv | [Link](https://arxiv.org/abs/2505.23839) | [Link](https://github.com/zaixizhang/GeneBreaker) ![GitHub Stars](https://img.shields.io/github/stars/zaixizhang/GeneBreaker) |
| 2025.05 | **BioReason: Incentivizing Multimodal Biological Reasoning within a DNA-LLM Model** | Bo Wang | [Link](https://github.com/bowang-lab) |  | DNA | arXiv | [Link](https://arxiv.org/abs/2505.23579) | [Link](https://github.com/bowang-lab/BioReason) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/BioReason) |
| 2025.05 | **CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells** |  |  |  | single cell RNA | Nature Communications | [Link](https://www.nature.com/articles/s41467-025-59926-5) | [Link](https://github.com/biomed-AI/CellFM) ![GitHub Stars](https://img.shields.io/github/stars/biomed-AI/CellFM) |
| 2025.04 | **Scaling Large Language Models for Next-Generation Single-Cell Analysis** | David van Dijk | [Link](https://github.com/vandijklab) |  | single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.14.648850v1) | [Link](https://github.com/vandijklab/cell2sentence) ![GitHub Stars](https://img.shields.io/github/stars/vandijklab/cell2sentence) |
| 2025.04 | **CellFlow enables generative single-cell phenotype modeling with flow matching** | Fabian J. Theis | [Link](https://github.com/theislab) |  | Perturbation single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.11.648220v1) | [Link](https://github.com/theislab/CellFlow) ![GitHub Stars](https://img.shields.io/github/stars/theislab/CellFlow) |
| 2025.02 | **Genome modeling and design across all domains of life with Evo 2** | Arc Institute(Brian L. Hie) | [Link](https://github.com/ArcInstitute) |  | DNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.18.638918v1) | [Link](https://github.com/arcinstitute/evo2) ![GitHub Stars](https://img.shields.io/github/stars/arcinstitute/evo2) |
| 2025.02 | **GENERator: A Long-Context Generative Genomic Foundation Model** |  |  |  | DNA | arXiv | [Link](https://arxiv.org/abs/2502.07272) | [Link](https://github.com/GenerTeam/GENERator) ![GitHub Stars](https://img.shields.io/github/stars/GenerTeam/GENERator) |
| 2025.02 | **scGPT-spatial: Continual Pretraining of Single-Cell Foundation Model for Spatial Transcriptomics** | Bo Wang | [Link](https://github.com/bowang-lab) |  | spatial single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.05.636714v1) | [Link](https://github.com/bowang-lab/scGPT-spatial) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/scGPT-spatial) |
| 2025.01 | **A foundation model of transcription across human cell types** |  |  |  | transcriptional regulation(ATAC-seq) | Nature | [Link](https://www.nature.com/articles/s41586-024-08391-z) | [Link](https://github.com/GET-Foundation) ![GitHub Stars](https://img.shields.io/github/stars/GET-Foundation/get_model) |
| 2025.01 | **GENA-LM: a new DNA language model for long sequences** |  |  |  | DNA | Nucleic Acids Research | [Link](https://academic.oup.com/nar/article/53/2/gkae1310/7954523) | [Link](https://github.com/AIRI-Institute/GENA_LM) ![GitHub Stars](https://img.shields.io/github/stars/AIRI-Institute/GENA_LM) |
| 2025.01 | **Function-Guided Conditional Generation Using Protein Language Models with Adapters** | Salesforce Research( Ali Madani) | [Link](https://github.com/salesforce) |  | Protein | arXiv | [Link](https://arxiv.org/abs/2410.03634) | [Link](https://github.com/Profluent-Internships/ProCALM) ![GitHub Stars](https://img.shields.io/github/stars/Profluent-Internships/ProCALM) |
| 2025.01 | **Simulating 500 million years of evolution with a language model** | EvolutionaryScale(Alexander Rives) | [Link](https://github.com/evolutionaryscale) |  | Protein | Science | [Link](https://www.science.org/doi/10.1126/science.ads0018) | [Link](https://github.com/evolutionaryscale/esm) ![GitHub Stars](https://img.shields.io/github/stars/evolutionaryscale/esm) |
| 2025.01 | **Predicting cell morphological responses to perturbations using generative modeling** | Fabian J. Theis, Mohammad Lotfollahi | [Link](https://github.com/theislab) |  | phenotype morphological responses to perturbation | Nature Communications | [Link](https://www.nature.com/articles/s41467-024-55707-8) | [Link](https://github.com/theislab/IMPA) ![GitHub Stars](https://img.shields.io/github/stars/theislab/IMPA) |
| 2024.11 | **Nucleotide Transformer: building and evaluating robust foundation models for human genomics** |  |  |  | DNA | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02523-z) | [Link](https://github.com/instadeepai/nucleotide-transformer) ![GitHub Stars](https://img.shields.io/github/stars/instadeepai/nucleotide-transformer) |
| 2024.10 | **Orthrus: Towards Evolutionary and Functional RNA Foundation Models** |  |  |  | RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.10.10.617658v1) | [Link](https://github.com/bowang-lab/Orthrus) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/Orthrus) |
| 2024.09 | **stFormer: a foundation model for spatial transcriptomics** |  |  |  | DNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.09.27.615337v6) | [Link](https://github.com/csh3/stFormer) ![GitHub Stars](https://img.shields.io/github/stars/csh3/stFormer) |
| 2024.08 | **BioRAG: A RAG-LLM Framework for Biological Question Reasoning** |  |  |  | Life science question answer (RAG) | arXiv | [Link](https://arxiv.org/abs/2408.01107) |  |
| 2024.07 | **DNA language model GROVER learns sequence context in the human genome** |  |  |  | DNA | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-024-00872-0) | [Link](https://zenodo.org/records/13374192) |
| 2024.06 | **scFoundation: Large-scale foundation model on single-cell transcriptomics** | Biomap research(Le Song, Xuegong Zhang) | [Link](https://github.com/biomap-research) |  | single cell RNA | Nature Communication | [Link](https://www.nature.com/articles/s41592-024-02305-7) | [Link](https://github.com/biomap-research/scFoundation) ![GitHub Stars](https://img.shields.io/github/stars/biomap-research/scFoundation) |
| 2024.06 | **Improving Medical Reasoning through Retrieval and Self-Reflection with Retrieval-Augmented Large Language Models** |  |  |  | biomedical question answer (RAG) | ISMB 24 | [Link](https://arxiv.org/abs/2401.15269) | [Link](https://github.com/dmis-lab/self-biorag) ![GitHub Stars](https://img.shields.io/github/stars/dmis-lab/self-biorag) |
| 2024.06 | **Multi-modal Transfer Learning between Biological Foundation Models** | InstaDeep(Guillaume Richard, Thomas Pierrot) | [Link](https://huggingface.co/InstaDeepAI) |  | Diverse Omics include DNA/RNA/Protein | arXiv | [Link](https://arxiv.org/abs/2406.14150) | [Link](https://huggingface.co/InstaDeepAI/isoformer) |
| 2024.05 | **Accurate structure prediction of biomolecular interactions with AlphaFold 3** | Google DeepMind(John M. Jumper) | [Link](https://github.com/google-deepmind) |  | Protein | Nature | [Link](https://www.nature.com/articles/s41586-024-07487-w) | [Link](https://github.com/google-deepmind/alphafold3) ![GitHub Stars](https://img.shields.io/github/stars/google-deepmind/alphafold3) |
| 2024.04 | **Masked Autoencoders for Microscopy are Scalable Learners of Cellular Biology** | Recursion(Berton Earnshaw) | [Link](https://www.rxrx.ai/) |  | microscopy data | CVPR 2024 Highlight | [Link](https://arxiv.org/abs/2404.10242) | [Link](https://www.rxrx.ai/phenom) |
| 2024.03 | **Caduceus: Bi-Directional Equivariant Long-Range DNA Sequence Modeling** |  |  |  | DNA | arXiv | [Link](https://arxiv.org/abs/2403.03234) | [Link](https://caduceus-dna.github.io/) ![GitHub Stars](https://img.shields.io/github/stars/kuleshov-group/caduceus) |
| 2024.03 | **Nicheformer: a foundation model for single-cell and spatial omics** | Fabian J. Theis | [Link](https://github.com/theislab) |  | single-cell and spatial omics | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.04.15.589472v1) | [Link](https://github.com/theislab/nicheformer) ![GitHub Stars](https://img.shields.io/github/stars/theislab/nicheformer) |
| 2024.02 | **scGPT: toward building a foundation model for single-cell multi-omics using generative AI** | Bo Wang | [Link](https://github.com/bowang-lab) |  | single cell RNA | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02201-0) | [Link](https://github.com/bowang-lab/scGPT) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/scGPT) |
| 2024.01 | **OmniNA: A foundation model for nucleotide sequences** |  |  |  | DNA/RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.01.14.575543v1) |  |
| 2024.01 | **xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein** | Biomap research(Le Song) | [Link](https://github.com/biomap-research) |  | Protein | arXiv | [Link](https://arxiv.org/abs/2401.06199) | [Link](https://github.com/biomap-research/xTrimoPGLM) ![GitHub Stars](https://img.shields.io/github/stars/biomap-research/xTrimoPGLM) |
| 2024.01 | **Geneverse: A Collection of Open-source Multimodal Large Language Models for Genomic and Proteomic Research** |  |  |  | DNA/Protein | EMNLP 2024 | [Link](https://openreview.net/forum?id=g1NplcWH7X) | [Link](https://github.com/HelloWorldLTY/Geneverse) ![GitHub Stars](https://img.shields.io/github/stars/HelloWorldLTY/Geneverse) |
| 2023.11 | **HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution** |  |  |  | DNA | NeurIPS 2023 | [Link](https://papers.neurips.cc/paper_files/paper/2023/file/86ab6927ee4ae9bde4247793c46797c7-Paper-Conference.pdf) | [Link](https://github.com/HazyResearch/hyena-dna) ![GitHub Stars](https://img.shields.io/github/stars/HazyResearch/hyena-dna) |
| 2023.10 | **ProGen2: Exploring the boundaries of protein language models** | Salesforce Research( Ali Madani) | [Link](https://github.com/salesforce) |  | Protein | Cell Systems | [Link](https://pubmed.ncbi.nlm.nih.gov/37909046/) | [Link](https://github.com/salesforce/progen) ![GitHub Stars](https://img.shields.io/github/stars/salesforce/progen) |
| 2023.06 | **DNABERT-2: Efficient Foundation Model and Benchmark For Multi-Species Genome** |  |  |  | DNA | ICLR 2024 | [Link](https://arxiv.org/abs/2306.15006) | [Link](https://github.com/MAGICS-LAB/DNABERT_2) ![GitHub Stars](https://img.shields.io/github/stars/MAGICS-LAB/DNABERT_2) |
| 2023.05 | **Transfer learning enables predictions in network biology** |  |  |  | General scRNA tasks, discovery of key network regulators and candidate therapeutic targets | Nature | [Link](https://www.nature.com/articles/s41586-023-06139-9) | [Link](https://huggingface.co/ctheodoris/Geneformer) |
| 2021.07 | **DNABERT: pre-trained Bidirectional Encoder Representations from Transformers for DNA-language** |  |  |  | DNA | Bioinformatics | [Link](https://academic.oup.com/bioinformatics/article/37/15/2112/6128680) | [Link](https://github.com/jerryji1993/DNABERT) ![GitHub Stars](https://img.shields.io/github/stars/jerryji1993/DNABERT) |

## AI Tools

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **GPTBioInsightor** |  |  |  | Single Cell annotation |  |  | [Link](https://github.com/huang-sh/GPTBioInsightor) ![GitHub Stars](https://img.shields.io/github/stars/huang-sh/GPTBioInsightor) |
| 2025.06 | **scExtract: leveraging large language models for fully automated single-cell RNA-seq data annotation and prior-informed multi-dataset integration** |  |  |  | scRNA downstream tasks | Genome Biology | [Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03639-x) | [Link](https://github.com/yxwucq/scExtract) ![GitHub Stars](https://img.shields.io/github/stars/yxwucq/scExtract) |
| 2025.04 | **Large Language Model Consensus Substantially Improves the Cell Type Annotation Accuracy for scRNA-seq Data** |  |  |  | Single Cell annotation | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.10.647852v1) | [Link](https://github.com/cafferychen777/mLLMCelltype) ![GitHub Stars](https://img.shields.io/github/stars/cafferychen777/mLLMCelltype) |
| 2025.04 | **SCassist: An AI Based Workflow Assistant for Single-Cell Analysis** |  |  |  | Single Cell tasks(cluster, annotation) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.22.650107v1) | [Link](https://github.com/NIH-NEI/SCassist) ![GitHub Stars](https://img.shields.io/github/stars/NIH-NEI/SCassist) |
| 2024.12 | **CASSIA: a multi-agent large language model for reference free, interpretable, and automated cell annotation of single-cell RNA-sequencing data** |  |  |  | Single Cell annotation | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.12.04.626476v2) | [Link](https://github.com/ElliotXie/CASSIA) ![GitHub Stars](https://img.shields.io/github/stars/ElliotXie/CASSIA) |

## Databases/Simulation

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.08 | **Drug Response Omics association MAp (DROMA, ÂçìÁéõ)** |  |  |  | Biomarker discovery, Drug prediction,(18 projects with 2,600+ samples and 56,000+ drugs, with full ecosystem) |  |  | [Link](https://github.com/mugpeng/DROMA) ![GitHub Stars](https://img.shields.io/github/stars/mugpeng/DROMA) |
| 2025.06 | **X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models** |  |  |  | Genome-wide Perturb-seq Datasets | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.11.659105v1) | [Link](https://doi.org/10.25452/figshare.plus.29190726) |
| 2025.04 | **scMultiSim: simulation of single-cell multi-omics and spatial data guided by gene regulatory networks and cell‚Äìcell interactions** |  |  |  | Simulation of single-cell multi-omics and spatial data | Nature Methods | [Link](https://www.nature.com/articles/s41592-025-02651-0) | [Link](https://github.com/ZhangLabGT/scMultiSim/) ![GitHub Stars](https://img.shields.io/github/stars/ZhangLabGT/scMultiSim) |
| 2025.03 | **RxRx3-core: Benchmarking drug-target interactions in High-Content Microscopy** | Recursion(Imran S. Haque) | [Link](https://www.rxrx.ai/) |  | microscopy perturbation data | ICLR 2025 | [Link](https://arxiv.org/abs/2503.20158v2) | [Link](https://www.rxrx.ai/datasets) |
| 2025.02 | **scBaseCount: an AI agent-curated, uniformly processed, and continually expanding single cell data repository** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) |  | preprocess scRNA data | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.27.640494v2) | [Link](https://github.com/ArcInstitute/SRAgent) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/SRAgent) |
| 2025.02 | **Tahoe-100M: A Giga-Scale Single-Cell Perturbation Atlas for Context-Dependent Gene Function and Cellular Modeling** | tahoebio(but in arc's repository) | [Link](https://github.com/ArcInstitute) |  | Drug perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.20.639398v3) | [Link](https://github.com/ArcInstitute/arc-virtual-cell-atlas) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/arc-virtual-cell-atlas) |
| 2024.06 | **Signals in the Cells: Multimodal and Contextualized Machine Learning Foundations for Therapeutics** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | Drug Discovery (Therapeutic science, 66 datasets) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.06.12.598655v3) | [Link](https://tdcommons.ai/) ![GitHub Stars](https://img.shields.io/github/stars/mims-harvard/TDC) |
| 2024.04 | **Three million images and morphological profiles of cells treated with matched chemical and genetic perturbations (Cell Painting Gallery)** | Broad Institute(Anne E. Carpenter) | [Link](https://github.com/broadinstitute) |  | phenotype morphological responses to perturbation | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02241-6) | [Link](https://github.com/broadinstitute/cellpainting-gallery/) ![GitHub Stars](https://img.shields.io/github/stars/broadinstitute/cellpainting-gallery) |
| 2024.01 | **scPerturb: harmonized single-cell perturbation data** |  |  |  | scRNA perturbation(44 public datasets, drugs and genes) | Nature Methods | [Link](https://www.nature.com/articles/s41592-023-02144-y) | [Link](https://github.com/sanderlab/scPerturb/) ![GitHub Stars](https://img.shields.io/github/stars/sanderlab/scPerturb) |

## Benchmarks

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.06 | **Fundamental Limitations of Foundation Models in Single-Cell Transcriptomics** |  |  |  | evaluate cell-type classification | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661767v1) |  |
| 2025.06 | **Cell-Eval(Predicting cellular responses to perturbation across diverse contexts with State)** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) |  | Perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661135v1) | [Link](https://github.com/ArcInstitute/cell-eval) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/cell-eval) |
| 2025.04 | **Zero-shot evaluation reveals limitations of single-cell foundation models** | Microsoft Research(Alex X. Lu) | [Link](https://github.com/microsoft) |  | cell type clustering and batch integration for scGPT and Geneformer | Genome Biology | [Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03574-x) | [Link](https://github.com/microsoft/zero-shot-scfoundation) ![GitHub Stars](https://img.shields.io/github/stars/microsoft/zero-shot-scfoundation) |
| 2024.12 | **PerturBench: Benchmarking Machine Learning Models for Cellular Perturbation Analysis** |  |  |  | Perturbation scRNA | NeurIPS 2024 | [Link](https://neurips.cc/virtual/2024/102911) | [Link](https://github.com/altoslabs/perturbench) ![GitHub Stars](https://img.shields.io/github/stars/altoslabs/perturbench) |

## Reviews

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **The generative era of medical AI** |  |  |  |  |  |  |  |
| 2025.05 | **Large Language Models in Bioinformatics: A Survey** |  |  |  | Bioinformatics foundation models | arXiv | [Link](https://arxiv.org/abs/2503.04490) |  |
| 2025.04 | **Model Context Protocol (MCP): Landscape, Security Threats, and Future Research Directions** |  |  |  | MCP for agents | arXiv | [Link](https://arxiv.org/abs/2503.23278) |  |
| 2025.04 | **A Survey of the Model Context Protocol (MCP): Standardizing Context to Enhance Large Language Models (LLMs)** |  |  |  | MCP for agents | Preprints | [Link](https://www.preprints.org/manuscript/202504.0245/v1) |  |
| 2024.10 | **Empowering biomedical discovery with AI agents** |  |  |  | Biomedical Agents | Cell | [Link](https://pubmed.ncbi.nlm.nih.gov/39486399/) |  |
| 2024.08 | **Transformers in single-cell omics: a review and new perspectives** |  |  |  | Single cell foundation models | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02353-z) |  |
| 2024.08 | **Language models for biological research: a primer** |  |  |  | biological foundation models | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02354-y) |  |
| 2024.07 | **Foundation models for bioinformatics** |  |  |  | Bioinformatics foundation models | Quantitative Biology | [Link](https://onlinelibrary.wiley.com/doi/10.1002/qub2.69) |  |

## Other Awesome Projects

| Title                                           | Project                                                      |
| ----------------------------------------------- | ------------------------------------------------------------ |
| **Awesome-LLMs-meet-genomes**                   | [Link](https://github.com/ychuest/Awesome-LLMs-meet-genomes) ![GitHub Stars](https://img.shields.io/github/stars/ychuest/Awesome-LLMs-meet-genomes) |
| **Awesome-Virtual-Cell**                        | [Link](https://github.com/Boom5426/Awesome-Virtual-Cell) ![GitHub Stars](https://img.shields.io/github/stars/Boom5426/Awesome-Virtual-Cell) |
| **Awesome-Phenotypic-Drug-Discovery**           | [Link](https://github.com/Boom5426/Awesome-Phenotypic-Drug-Discovery) ![GitHub Stars](https://img.shields.io/github/stars/Boom5426/Awesome-Phenotypic-Drug-Discovery) |
| **Awesome-LLM-Scientific-Discovery**            | [Link](https://github.com/HKUST-KnowComp/Awesome-LLM-Scientific-Discovery) ![GitHub Stars](https://img.shields.io/github/stars/HKUST-KnowComp/Awesome-LLM-Scientific-Discovery) |
| **Awesome LLM Agents for Scientific Discovery** | [Link](https://github.com/zhoujieli/Awesome-LLM-Agents-Scientific-Discovery) ![GitHub Stars](https://img.shields.io/github/stars/zhoujieli/Awesome-LLM-Agents-Scientific-Discovery) |
| **Awesome Papers on Agents for Science**        | [Link](https://github.com/OSU-NLP-Group/awesome-agents4science) ![GitHub Stars](https://img.shields.io/github/stars/OSU-NLP-Group/awesome-agents4science) |
| **awesome-single-cell**                         | [Link](https://github.com/seandavi/awesome-single-cell) ![GitHub Stars](https://img.shields.io/github/stars/seandavi/awesome-single-cell) |
| **Awesome-Bioinformatics**                      | [Link](https://github.com/danielecook/Awesome-Bioinformatics) ![GitHub Stars](https://img.shields.io/github/stars/danielecook/Awesome-Bioinformatics) |
| **awesome**                                     | [Link](https://github.com/sindresorhus/awesome) ![GitHub Stars](https://img.shields.io/github/stars/sindresorhus/awesome) |

## Others

# Ref